storage/byodb/external-buckets/index.md

```diff
@@ -0,0 +1,105 @@
+If you later add additional tables or views, you will need to manually refresh the external bucket using the **Refresh** action in the bucket detail to make them visible in Keboola Connection. 
+
+{: .image-popup}
+![Bucket refresh](/storage/byodb/external-buckets/figures/4.png)
+
+## Using an External Bucket
+
+External buckets are not part of a normal [input mapping](transformations/mappings/#input-mapping). They are accessible via the [read-only input mapping](/transformations/mappings/#read-only-input-mapping). 
+Keep in mind that external buckets cannot be used in an [output mapping](transformations/mappings/#output-mapping) as they are not writable.
+
+### External Bucket in a Snowflake SQL Transformation
+
+Because external buckets are not part of a normal [input mapping](transformations/mappings/#input-mapping), they are not copied into your transformation workspace. 
+You need to reference them in you transformation using a fully-qualified name.
+
+In the following example, you will create an external bucket called `emea_sales` that references the `sales_emea`schema in the database `REPORTING`. The schema contains a table called `users`. 
+You want to create a new table `MQL_USERS` that contains only users sourced from marketing qualified leads. You can do that using the following SQL:
+
+```sql
+CREATE TABLE "MQL_USERS" AS SELECT * 
+FROM "REPORTING"."sales_emea"."users"
+WHERE "source" = 'mql';
+```
+*Note: The query uses the **fully-qualified name** of the table in the `FROM` clause.*
+
+### External Bucket in a BigQuery SQL Transformation
+
+In BigQuery, the external bucket is mapped to an actual dataset, `emea_sales` (the name you filled in the dialog), in your project – in this case, project `sapi-9752`. 
+You can reference the contents of the dataset in your SQL transformation using a fully-qualified name. 
+
+*Note: The dataset name is _the one you provided in the dialog_, **not** that of the original dataset created in BigQuery. However, there are no technical limitations; they can have the same name.* 
+
+## Removing an External Bucket
+
+Removing an external bucket is as simple as removing any other bucket. Simply delete it in the UI or via API. The bucket will be removed from the project, but the schema in the database will remain untouched. 
+If you wish to remove the schema, you will need to do so manually in your warehouse. You may want to revoke the grants to Keboola Connection to ensure that Keboola no longer has access to it.
+
+## Usage Recommendations
+
+* Use external buckets to consume data in your warehouse produced by 3rd-party tools outside of Keboola.
+* Use external buckets to access data in table-like structures that are not directly supported by Keboola (e.g., views or external tables).
+* **Do not** use external buckets to load data into Keboola Connnection from services that already have an existing component.
```

Comment:
nooo, vidim ty duvody dole, ale asi bych to takto nedoporucoval, spis bych rekl nasledujici

Be aware that using External buckets might have following consequences:

Reply:
imho proste je validni, ze chces do kebooly prinest data skrze EB.

Reply:
Změkčil jsem to na to, že to nedoporučujeme...